GoogleMulticastAppDelegate/Sources/Public/GoogleUtilities/GULMulticastAppDelegate.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#if TARGET_OS_IOS || TARGET_OS_TV
+
+#import <UIKit/UIKit.h>
+
+#define GULApplication UIApplication
```

Comment:
@paulb777 temporarily copy over GULApplication.h from AppDelegateSwizzler. I wonder if it's possible to create a public header folder depended by GoogleUtilities/AppDelegateSwizzler and GULMulticastAppDelegate.

Reply:
It might be cleaner to make a GULMApplication.h and then we can cleanly delete everything about the old swizzler in a breaking change update.

Reply:
Discussed offline, will let multicastAppDelegate depend on AppDelegateSwizzler since all users are using swizzling. Will move common code to multicastAppDelegate once we deprecate and remove swizzling.